### PR TITLE
Changes browser tab title to 'Eviction Guide.'

### PR DESCRIPTION
**Pull request by mishs**

### CODE REVIEW

**Comment by PiDelport** [comment]
@mishs: Does this replace PR #1?

File: src/components/BaseLayout/index.jsx [hunk 1]
```diff
@@ -27,6 +27,10 @@ const BaseLayout = ({ children, classes }) => {
 
   return (
     <MuiThemeProvider {...{ theme }}>
+        <Helmet>        
+         <meta charSet="utf-8" />
```
Comment (PiDelport) [inline src/components/BaseLayout/index.jsx]: This charset declaration seems to be unnecessary?

```suggestion
```

**Comment by PiDelport** [comment]
Card: https://trello.com/c/D2A8IWn4/4-change-evictionsorgza-page-title-to-eviction-guide

**Comment by mishs** [comment]
> @mishs: Does this replace PR #1?

Yes.

**Comment by PiDelport** [comment]
@mishs: Cool; remember to close PRs if they're no longer active. :)

**Comment by PiDelport** [comment]
@mishs: Are you happy with the change?

File: src/components/BaseLayout/index.jsx (hunk 1 above)
Comment (mishs) [reply src/components/BaseLayout/index.jsx]: Yes it's not necessary

**Pull request merged**